syntax = "proto2";

package rv.protos;

import "google/protobuf/struct.proto";

message RasterTransformerConfig {
    required string transformer_type = 1;

    oneof raster_transformer_config {
        // Configuration for StatsTransformer
        // File with stats generated by compute_stats command. Used to
        // convert non-uint8 raster values to uint8.
        string stats_uri = 4;

        // Configuration for custom transformers
        google.protobuf.Struct custom_config = 5;
    }
}
